lib/ex_docker_build.ex

```diff
@@ -201,6 +201,39 @@ defmodule ExDockerBuild do
     end
   end
 
+  @spec inspect_volume(String.t()) :: :ok | {:error, any()}
+  def inspect_volume(volume_name) do
+    case DockerRemoteAPI.inspect_volume(volume_name) do
+      {:ok, %{status_code: 200}} ->
+        :ok
+
+      {:ok, %{body: body, status_code: _}} ->
+        {:error, body}
+
+      {:error, %{reason: reason}} ->
+        {:error, reason}
+    end
+  end
+
+  @spec delete_volume(String.t()) :: :ok | {:error, any()}
+  def delete_volume(volume_name) do
```

Comment:
you need to add tests for this function, one way of doing it is adding support for listing volumes with filters https://docs.docker.com/engine/api/v1.30/#operation/VolumeList

Reply:
You are right.